src/pages/Map.js

```diff
@@ -16,7 +16,6 @@ const containerStyle = {
 };
 
 function Map() {
-  const [activeMarker, setActiveMarker] = useState(null);
   const [markers, setMarkers] = useState([]);
   const { isLoaded, loadError } = useLoadScript({
     googleMapsApiKey: "AIzaSyBQ6AXhN1dWqYH5pjf8zuIoUyfTb1j0bAY",
```

Comment:
so API keys should typically be in a completely other file (something like keys or tokens or whatever) and we retrieve from that file.